src/java/org/apache/cassandra/cql3/statements/schema/CreateTableStatement.java

```diff
@@ -543,14 +550,17 @@ public String table()
             return name.getName();
         }
 
-        public void addColumn(ColumnIdentifier column, CQL3Type.Raw type, boolean isStatic, ColumnMask.Raw mask)
+        public void addColumn(ColumnIdentifier column, CQL3Type.Raw type, boolean isStatic, ColumnMask.Raw mask, ColumnConstraints.Raw constraints)
         {
-
             if (null != rawColumns.put(column, new ColumnProperties.Raw(type, mask)))
                 throw ire("Duplicate column '%s' declaration for table '%s'", column, name);
 
             if (isStatic)
                 staticColumns.add(column);
+            if (null == constraints)
+                columnConstraints.put(column, new ColumnConstraints.Noop());
```

Comment:
`new ColumnConstraints.Noop()` can be singleton, applicable everywhere this occurs